packages/terra-date-picker/src/DateUtil.js

```diff
@@ -499,7 +499,14 @@ class DateUtil {
    * @return {moment} The incremented moment date
    */
   static incrementDateByDay(date, format) {
-    return moment(date, format, true).add(1, 'd').format('YYYY-MM-DD');
+    const momentDate = moment(date, format, true).add(1, 'd');
+    if (momentDate.year() < Number(DateUtil.MIN_YEAR)) {
+      return DateUtil.MIN_DATE;
```

Comment:
It's not a big deal but I don't think it will ever get in this scenario because the year should never be 1899 to being with.

Reply:
We shouldn't get into the scenario right now, but I would prefer leaving it in for protecting future uses of method just in case.